### PR TITLE
Update References from Twitter to X (Fixes #168)

### LINKED ISSUE #168: Update Twitter links to X links
Update Twitter.com links to X.com links, throughout the codebase.
Especially the social section.

### PR DESCRIPTION
**Pull request by BishalJena**
This pull request updates all references from Twitter to X in the project to align with the rebranding. 
Changes include:

Updated links and mentions in the documentation #168 
Additionally, the .DS_Store file has been removed, and .gitignore has been updated to prevent it from being tracked in the future.

By merging this PR, we ensure that all references are consistent with the new branding and improve the repository's cleanliness.
Fixes: #168

### CODE REVIEW

**Comment by BishalJena** [comment]
@deluan kindly merge this PR.

File: config.toml [hunk 1]
```diff
@@ -152,10 +152,10 @@ no = 'Sorry to hear that. Please <a href="https://github.com/navidrome/website/i
 	icon = "fab fa-discord"
         desc = "Chat with other users and project developers"
 [[params.links.user]]
-	name = "Twitter"
-	url = "https://twitter.com/navidrome"
-	icon = "fab fa-twitter"
-        desc = "Follow us on Twitter"
+	name = "X"
+	url = "https://x.com/navidrome"
+	icon = "fab fa-x"
```
Comment (deluan) [inline config.toml:157]: The X logo is actually fa-x-twitter (see: https://fontawesome.com/icons/x-twitter?f=brands&s=solid)

**Review by deluan — CHANGES_REQUESTED** [review]
Thanks for this. Please see my comment below.

File: config.toml (hunk 1 above)
Comment (BishalJena) [reply config.toml]: made the change @deluan 
merge if passes the check.

**Comment by deluan** [comment]
Seems that the version of Docsy we use is old and does not include the new X logo:

Screenshot 2024-08-02 at 4 21 01 PM

Let me know if you can do this as part of this PR. If not , I'll take a look later. 

Thanks!

**Comment by BishalJena** [comment]
i'd just like to submit my icon change for now.
maybe a new issue can be raised to solve the docsy update thing and someone else can look into it.
thanks!

**Pull request merged**